contrib/builder/deb/generate.sh

```diff
@@ -92,6 +104,40 @@ for version in "${versions[@]}"; do
 
 	echo >> "$version/Dockerfile"
 
+	# debian jessie & ubuntu trusty/vivid do not have a libseccomp.a for compiling static dockerinit
+	# ONLY install libseccomp.a from source, this can be removed once dockerinit is removed
+	case "$suite" in
+		jessie|trusty|vivid)
+			awk '$1 == "ENV" && $2 == "SECCOMP_VERSION" { print; exit }' ../../../Dockerfile >> "$version/Dockerfile"
+			cat <<-EOF >> "$version/Dockerfile"
+			RUN buildDeps=' \
+				automake \
+				libtool \
+			' \
+			&& set -x \
+			&& apt-get update && apt-get install -y \$buildDeps --no-install-recommends \
+			&& rm -rf /var/lib/apt/lists/* \
+			&& export SECCOMP_PATH=\$(mktemp -d) \
+			&& git clone https://github.com/seccomp/libseccomp.git "\$SECCOMP_PATH" \
```

Comment:
Since we don't care about verbose warnings from `git` in this build output, couldn't we change this to something like `git clone -b "$SECCOMP_VERSION" --depth 1 ...` so that it goes slightly faster?